app/lib/import_export.rb

```diff
@@ -345,10 +672,10 @@ def latest_transfers(patients)
         ]
   end
 
-  # 2 Latest laboratories of each patient
-  def latest_laboratories(patients)
+  # Gets a hash of the 2 latest laboratories of each patient
+  def get_patients_laboratories(patients)
     latest_labs = Hash[patients.pluck(:id).map { |id| [id, {}] }]
-    Laboratory.where(patient_id: patients.pluck(:id)).order(report: :desc).each do |lab|
+    Laboratory.where(patient_id: patients.pluck(:id)).order(specimen_collection: :desc).each do |lab|
```

Comment:
Extract `patients.pluck(:id)` into a variable so you only do that once instead of twice.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/1f13143f8968feff3c7e9ada6139b47e5cd4ecff#diff-c34521fcd0fdc2034d456886aae8fd35299fe908ab88d8e49c60d0b4d39ab26bR677